nextflow_schema.json

```diff
@@ -1457,6 +1462,37 @@
             },
             "fa_icon": "fas fa-angle-double-right"
         },
+        "cazyme_dbcan": {
+            "title": "dbCAN",
+            "type": "object",
+            "description": "Carbohydrate-active enzyme annotation based on pre-defined HMM models.\n\nFor more information check the dbCAN [documentation](https://run-dbcan.readthedocs.io/en/latest)",
+            "default": "",
+            "properties": {
+                "cazyme_skip_dbcan": {
+                    "type": "boolean",
+                    "description": "Skip dbCAN during the CAZyme screening.",
+                    "fa_icon": "fas fa-ban"
+                },
+                "dbcan_skip_cgc": {
+                    "type": "boolean",
+                    "description": "Skip CGC during the dbCAN screening.",
```

Comment:
Please add a long-form ('help_text') explaining what CGC is, and same for substrate down below...

As an outsider, I don't know what CGC/substrates are in this context :sweat_smile: are they `db_can` specific terms?

Reply:
Still missing